frontend/server/src/main/java/com/amazonaws/ml/mms/metrics/Metric.java

```diff
@@ -45,6 +45,15 @@
     @SerializedName("HostName")
     private String hostName;
 
+    public Metric() {}
+
+    public Metric(String metricName, String value, String unit, List<Dimension> dimensions) {
```

Comment:
Metrics(String name, String value, String unit, Dimension.. dimensions) is better.